embedchain/vectordb/lancedb.py

```diff
@@ -0,0 +1,309 @@
+from typing import Any, Dict, List, Optional, Union
+
+import pyarrow as pa
+
+try:
+    import lancedb
+except ImportError:
+    raise ImportError("LanceDB is required. Install with `pip install embedchain[lancedb]`") from None
```

Comment:
`pip install embedchain[lancedb]` would be actually `pip install "embedchain[lancedb]"`